js/apps/admin-ui/src/realm-settings/GeneralTab.tsx

```diff
@@ -94,7 +94,7 @@ export const RealmSettingsGeneralTab = ({
               required: { value: true, message: t("required") },
               pattern: {
                 value: /^[a-zA-Z0-9-_]+$/,
-                message: t("realm:invalidRealmName"),
+                message: t("realm:invalidRealmId"),
```

Comment:
Does passing "realm:invalidRealmId" refer to something other than "invalidRealmId"?